tests/unit/hook_packages/terraform/hooks/prepare/test_translate.py

```diff
@@ -1136,5 +1137,9 @@ def test_module_contains_unresolvables(self, log_mock, mapping_mock):
         _check_unresolvable_values(module, tf_module)
 
         log_mock.warning.assert_called_with(
-            Colored().yellow("\nUnresolvable attributes discovered in project, run terraform apply to resolve them.\n")
+            Colored().color_log(
+                msg="\nUnresolvable attributes discovered in project, " "run terraform apply to resolve them.\n",
```

Comment:
nit: cleaning the double quote thing that the formatters sometimes leave for some reason